web/blog/parquet-and-feather-data-engineering-woes/index.md

```diff
@@ -0,0 +1,232 @@
+---
+draft: true
+title: "Parquet & Feather: Data Engineering Woes"
+authors:
+  - dispanser
+  - mavam
+date: 2023-01-10
+image: arrow-schema-conversion.light.png
```

Comment:
The carousel on the frontpage of the docs checks if this path begins with a slash, and otherwise refuses to render the image. Not sure if the bug is in the carousel or here.

Reply:
I removed this. Unless an image is in `/static/img` and can be referenced via `/img/...`, the image will not be properly displayed because the name gets mangled to something internal.